rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/NodeSourceWindow.java

```diff
@@ -687,7 +687,7 @@ private List<FormItem> getPrefilledFormItems(PluginDescriptor plugin) {
                 formItemsForField.add(new TextAreaItem(plugin.getPluginName() + pluginField.getName(),
                                                        pluginField.getName()) {
                     {
-                        setWrap(TextAreaWrap.OFF);
+                        setWrap(TextAreaWrap.HARD);
```

Comment:
you should not use hard wrap (insert line break) on fields like java options -Doption=value. This will break the command line.

use TextAreaWrap.SOFT instead

Reply:
Additionnally, there is a risk that changing java options to text area could confuse users in believing they can write separate lines. But I don't know for now how to prevent this.

Reply:
One possible way to prevent this risk is in the various infrastructure implementation in the configure method.

for example:

this.javaOptions = parameters[index++].toString().replace(System.lineSeparator(), " ");